src/components/DropDownButton/index.tsx

```diff
@@ -203,12 +209,12 @@ class DropDownButton extends React.Component<
 		return (
 			<Wrapper
 				className={className}
-				{...props}
+				style={{ background: 'none' }}
+				{...omit(props, 'bg')}
 				ref={node => (this.dropdownNode = node)}
 			>
 				{joined ? (
 					<Toggle
-						{...props}
```

Comment:
I haven't looked into the code-base for this but are you sure that this component is never used by someone who is relying on a prop being passed through here? This _looks_ like a breaking change to me (albeit probably a very insignificant one).

Reply:
I agree, I'll return this prop. I think it got removed during testing.